MAUI/Popup/popup-positioning.md

```diff
@@ -0,0 +1,266 @@
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+<ContentPage.Content>       
+  <StackLayout VerticalOptions="Start" HorizontalOptions="Start" Padding="20">
+    <Button x:Name="clickToShowPopup" Text="ClickToShowPopup"
+            VerticalOptions="Center" HorizontalOptions="Start"
+            Clicked="ClickToShowPopup_Clicked" />
+    <sfPopup:SfPopup x:Name="sfPopup">
+    </sfPopup:SfPopup>
+  </StackLayout>
+</ContentPage.Content>
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+private void ClickToShowPopup_Clicked(object sender, EventArgs e)
+{
+  // Shows SfPopup at the bottom of the button and with absolute relative position.
+	sfPopup.ShowRelativeToView(clickToShowPopup, PopupRelativePosition.AlignBottom, 50, 50);
+}
+{% endhighlight %}
+{% endtabs %}
+
+![Absolute positioning of Popup](Images/popup-positioning/maui-popup-absolute-relative-positioning.png)
+
+You can pass both negative and positive values as parameters to the SfPopup.ShowRelativeToView(View, RelativePosition, x-position, y-position). The popup will be positioned by considering the relative position as (0, 0) the center point. For example, if you have set the RelativePosition as RelativePosition.BottomRight and RelativeView as a button, bottom right corner of the button will be considered as the 0, 0 point and a negative x-position value will place the popup to the left of that point and a positive x-position value will place the popup to the right of that point. The same applies for y-position also.
+
+### Show relative to view in MVVM
+
+To open the SfPopup relative to a view in MVVM assign values to the `SfPopup.RelativeView` and `SfPopup.RelativePosition` properties and use the `SfPopup.IsOpen` property to open or close the popup using binding.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:sfPopup="clr-namespace:Syncfusion.Maui.Popup;assembly=Syncfusion.Maui.Popup"
+             xmlns:local="clr-namespace:PopupMauiPositioning"
+             x:Class="PopupMauiPositioning.MainPage">
+  <ContentPage.BindingContext>
+    <local:ViewModel/>
+  </ContentPage.BindingContext>
+  <ContentPage.Content>       
+    <StackLayout>
+      <Button x:Name="clickToShowPopup" Text="ClickToShowPopup" Margin="0,30,0,30"
+              HorizontalOptions="Center"
+              Clicked="ClickToShowPopup_Clicked" />
+      <Label x:Name="relativeView" Text="Showing Popup at relative position in MVVM" 
+             VerticalOptions="StartAndExpand" HorizontalOptions="Center" 
+             HorizontalTextAlignment="Center" FontFamily="Roboto" Padding="20"
+             LineBreakMode="WordWrap" FontSize="14" BackgroundColor="#6750A4" TextColor="White"/>
+      <sfPopup:SfPopup x:Name="sfPopup" IsOpen="{Binding DisplayPopup}"

```

Comment:
<sfPopup:SfPopup x:Name="sfPopup" 
                               IsOpen="{Binding DisplayPopup}".
                               
Modify like above and hightlight "IsOpen" code